### DISCUSSION TITLE
Keep getting "Cannot read properties of null (reading 'addEventListener')" when trying to add panning

### DISCUSSION THREAD

**Discussion by BLE77** [google/model-viewer#2988] [Q&A]
I even made a separate JS and put it inside an on load function. I am also using the poster method with interaction and i was thinking it could be a problem? but im kinda just a little lost pls dont hate me if this is a dumb and easy question, im new to all this. https://github.com/BLE77/exp2

**Comment by elalish** [comment]
Please post a deployed URL of the website  where we can see the console errors. I  don't have time to  build anyone's repos 😂

It sounds like you just haven't retrieved the  right  element yet.

**Reply by BLE77** [reply]
https://ble77.github.io/exp2/  my bad!

**Reply by elalish** [reply]
Yep, looks like just a typo in your id:  `'pan-demo` should be `pan-demo`.

**Reply by BLE77** [reply]
ooo ty ty, next time ill scan harder! I definitely tried my best to avoid asking , sorry for bringing in such a silly problem
